lib/peep.rb

```diff
@@ -0,0 +1,38 @@
+require 'pg'
+
+class Peep
+
+  def self.all
+
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
I would suggest to move the if else statement in a script that runs when the app loads

Create a file, require the pg gem, and add this if statement.
Require the file at the top of your controller
Delete the if statements